frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -154,6 +159,66 @@ const SessionContainer = styled.div`
   }
 `;
 
+const SessionCardContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  min-width: 100%;
+  grid-gap: 32px;
+
+  @media (max-width: 900px) {
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 24px;
+  }
+`;
+
+interface SessionCompletionProp {
+  isComplete: boolean;
+}
+
+const SessionBox = styled.div`
+  padding: 40px;
+  border-radius: 12px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  @media (max-width: 900px) {
+    padding: 32px;
+  }
+
+  ${({ isComplete }: SessionCompletionProp) => {
+    if (isComplete) {
+      return `
+    background: ${SKY} !important;
+  `;
+    } else {
+      return `
+    background: ${INCOMPLETE_GRAY} !important; 
+  `;
+    }
+  }};
```

Comment:
Can we convert this to be a bit smaller, like the suggestion? (can't promise if the syntax is correct or not, but that vibe)
```suggestion
  background: $({isComplete}: SessionCompletionProp) => isComplete ? SKY : INCOMPLETE_GRAY} !important
```

Reply:
Yes, thanks!